.travis.yml

```diff
@@ -5,6 +5,8 @@ matrix:
   include:
   - jdk: openjdk11
     env: TRIPLEA_RELEASE=true
+  - jdk: openjdk12
+    env: TRIPLEA_RELEASE=true
```

Comment:
Only one jdk should have the triplea_release flag set to true, so this setting value would want to be flipped to 'false'. 

Checking usages:
```
~/work/triplea$ grep -r "TRIPLEA_RELEASE"
.travis.yml:    env: TRIPLEA_RELEASE=true
.travis.yml:    condition: "$TRIPLEA_RELEASE = true"
```
Looking at that file, seems we would get artifacts built when that is set to true for the corresponding JDK. We'd only want one set to be generated. For now we're targeting JDK11. To jump to JDK12 we'd need to update the install4j bundled JREs to be version 12 and fix any build issues with java 12. The bunlded JRE is used with the game installer, hence the extra coordination before we can switch java versions. The jump to 11 is pretty recent and the latest release is still on java 8.